app/models/clearCache.js

```diff
@@ -134,6 +134,14 @@ module.exports = async (instance, method) => {
       return removeKeys(`/reports/${report.ident}/msi`);
     case 'signatures':
       return removeKeys(`/reports/${report.ident}/signatures`);
+    case 'copyVariants':
+    case 'expressionVariants':
+    case 'kbMatches':
```

Comment:
what's happening with kbMatches and genes in this pr?

Reply:
Genes, kb-matches as well as the opposite variant (expression for copy and vice versa) are eager loaded for both routes (copy-variants and expression-variants). So, if any of these are updated for the given report is very likely the cache now contains incorrect values and needs to be cleared.